src/Modules/CalcPerform.lua

```diff
@@ -3141,11 +3141,14 @@ function calcs.perform(env, avoidCache)
 
 			-- Set trigger rate
 			local hits_per_cast = env.player.mainSkill.skillPart == 2 and env.player.mainSkill.activeEffect.srcInstance.skillStageCount or 1
-			output.EffectiveSourceRate = hexCastRate * hits_per_cast
+			output.EffectiveSourceRate = hexCastRate
```

Comment:
I don't think hits_per_cast (or rather, the "stages" input) is included in the calculation at all any more, is it?

Also, I noticed that I used the wrong naming convention for hits_per_cast.